loader/test/support/schemas.js

```diff
@@ -44,6 +44,7 @@ const capacitySchema = {
     unavailable_count: { type: "integer", minimum: 0 },
     products: productsSchema,
     dose: { type: "string" },
+    booking_url: { type: "string", format: "uri" },
```

Comment:
It turns out other sources have been including this, but didn’t have test data that covered it. 🤷